dataactbroker/handlers/accountHandler.py

```diff
@@ -551,120 +553,121 @@ def changeStatus(self,system_email):
-        if(user.email == None):
+        if user.email is None:
             return JsonResponse.error(ResponseException("User does not have a defined email",StatusCode.INTERNAL_ERROR),StatusCode.INTERNAL_ERROR)
 
         #check if the user is waiting
-        if(self.interfaces.userDb.checkStatus(user,"awaiting_approval")):
-            if(requestDict.getValue("new_status") == "approved"):
+        if self.interfaces.userDb.checkStatus(user,"awaiting_approval"):
+            if request_dict.getValue("new_status") == "approved":
                 # Grant agency_user permission to newly approved users
                 self.interfaces.userDb.grantPermission(user,"agency_user")
                 link=  AccountHandler.FRONT_END
-                emailTemplate = { '[URL]':link,'[EMAIL]':system_email}
-                newEmail = sesEmail(user.email, system_email,templateType="account_approved",parameters=emailTemplate,database=self.interfaces.userDb)
-                newEmail.send()
-            elif (requestDict.getValue("new_status") == "denied"):
-                emailTemplate = {}
-                newEmail = sesEmail(user.email, system_email,templateType="account_rejected",parameters=emailTemplate,database=self.interfaces.userDb)
-                newEmail.send()
+                email_template = { '[URL]':link,'[EMAIL]':system_email}
+                new_email = sesEmail(user.email, system_email,templateType="account_approved",parameters=email_template,database=self.interfaces.userDb)
+                new_email.send()
+            elif request_dict.getValue("new_status") == "denied":
+                email_template = {}
+                new_email = sesEmail(user.email, system_email,templateType="account_rejected",parameters=email_template,database=self.interfaces.userDb)
+                new_email.send()
         # Change user's status
-        self.interfaces.userDb.changeStatus(user,requestDict.getValue("new_status"))
+        self.interfaces.userDb.changeStatus(user,request_dict.getValue("new_status"))
         return JsonResponse.create(StatusCode.OK,{"message":"Status change successful"})
 
-    def listUsers(self):
+    def list_users(self):
         """ List all users ordered by status. Associated request body must have key 'filter_by' """
-        requestDict = RequestDictionary(self.request, optionalRequest=True)
-        user_status = requestDict.getValue("status") if requestDict.exists("status") else "all"
+        request_dict = RequestDictionary(self.request, optionalRequest=True)
+        user_status = request_dict.getValue("status") if request_dict.exists("status") else "all"
         sess = GlobalDB.db().session
 
-        user = self.interfaces.userDb.getUserByUID(LoginSession.getName(flaskSession))
-        isAgencyAdmin = self.userManager.hasPermission(user, "agency_admin") and not self.userManager.hasPermission(user, "website_admin")
+        user = sess.query(User).filter(User.user_id == LoginSession.getName(flaskSession)).one()
```

Comment:
That sounds like a good idea, we can bring that up at some point (as you said, in a different PR/ticket)